.eslintrc.js

```diff
@@ -12,16 +12,13 @@ module.exports = {
         "import/no-extraneous-dependencies": [0, { devDependencies: false }],
       },
     },
-    {
-      files: ["*/**/ducks/*.ts"],
-      rules: {
-        "no-param-reassign": "off",
-      },
-    },
   ],
   rules: {
-    "no-console": 0,
```

Comment:
Looks like we might need to put the `no-console` rule back, it's not happy about it in some places.